File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/RedwoodBow.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Bow;

/* JADX INFO: loaded from: classes3.dex */
public class RedwoodBow extends Bow {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.weapon_bow_redwood_bow_name;
        this.idDescription = R.string.weapon_bow_redwood_bow_description;
        this.idImage = R.drawable.redwood_bow;
        this.price = 123L;
        this.constitution = 4;
        this.dexterity = 12;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/RedwoodBow.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Bow

class RedwoodBow : Bow() {
    override fun configureProperties() {
        idName = R.string.weapon_bow_redwood_bow_name
        idDescription = R.string.weapon_bow_redwood_bow_description
        idImage = R.drawable.redwood_bow
        price = 123L
        constitution = 4
        dexterity = 12
    }
}
